### PR TITLE
update all.json

### PR DESCRIPTION
**Pull request by ical10**
|Site|Urlscan|Screenshot|
|--|--|--|
|collablandsync.org|https://urlscan.io/result/0fae9eae-042c-484b-b3a1-49fb2f56cf67/|https://user-images.githubusercontent.com/14356826/180135229-54c43ad7-850b-4977-ae35-6fbae26c3913.png|
|dapp-supporteam.netlify.app|https://urlscan.io/result/ebbfcea0-87a8-48a8-8023-acdb03e460fd/|https://user-images.githubusercontent.com/14356826/180135255-275148ca-352f-40f5-8dcb-b74c272ba442.png|
|gkrypto.online|https://urlscan.io/result/8f446bc6-0882-4855-89ed-4d710919f767/|https://user-images.githubusercontent.com/14356826/180135245-e35dade0-108c-43de-8504-594fb86af5d3.png|
|interfacevalidation.on.fleek.co|NA|https://user-images.githubusercontent.com/14356826/180135253-df93f32e-3322-4cc6-b1ef-558f1151cf88.png|
|set-protocol.space|https://urlscan.io/result/14aba7c2-8c3f-44e5-961e-e5824f48ac1a/dom/|https://user-images.githubusercontent.com/14356826/180135242-3ab84335-47fa-4864-84a1-0b8ea1c3d695.png|
|synnex-network.xyz|https://urlscan.io/result/1abae5e2-2b98-46ac-b90f-c9357a38101a/dom/|https://user-images.githubusercontent.com/14356826/180135248-71cf9d38-82ad-4669-b54d-327e0bda4760.png|

### CODE REVIEW

**Comment by ical10** [comment]
@jacogr trying to add `interfacevalidation.on.fleek.co` but the test said `added urls › has no entries for allowed top-level domains`
what does it mean?

**Comment by jacogr** [comment]
Lemme check that one - the issue is that previously we had urls like `storageapi.fleek.co` which blocked the whole domain - and it has valid uses. So at this point anything `*.fleek.co` is white-listed.

For now drop that one, I'm guessing we need to make an exception (somehow) for `*.on.fleek.co`, i.e. those may be user URLs.

**Comment by jacogr** [comment]
Putting in a work-around and will add that URL in https://github.com/polkadot-js/phishing/pull/1871

**Comment by ical10** [comment]
Okay, dropping `interfacevalidation.on.fleek.co`

File: all.json [hunk 1]
```diff
@@ -1,6 +1,31 @@
...
+		"collablandd.com",
+		"collablandsecuredconnect.com",
+		"metamask-walletsynfix-247.square.site",
+		"dappsreapprove.xyz",
+		"coinsolutionnetwork.live",
+		"apikucoinconnect.com",
+		"filsfixmimas.com",
+		"node-debug.com",
+		"dappsvalidator.netlify.app",
+		"atomic.cryptlinkaffix.online",
```
Comment (jacogr) [inline all.json:21]:
```suggestion
		"atomic.cryptlinkaffix.online",
		"cryptlinkaffix.online",
```

Root level is also problematic.

**Review by jacogr — APPROVED** [review]
Thank you!

**Comment by ical10** [comment]
thanks jaco!

**Comment by ical10** [comment]
Note: `interfacevalidation.on.fleek.co` was added in #1871 instead

**Pull request merged**